homeassistant/components/calendar/__init__.py

```diff
@@ -180,6 +191,171 @@ def _get_date(date):
         self._cal_data['offset_time'] = offset_time
         self._cal_data['location'] = self.data.event.get('location', '')
         self._cal_data['description'] = self.data.event.get('description', '')
+        self._cal_data['url'] = self.data.event.get('url', '')
+        self._cal_data['start'] = start
+        self._cal_data['end'] = end
+        self._cal_data['all_day'] = 'date' in self.data.event['start']
+
+
+# pylint: disable=too-many-instance-attributes
+class AsyncCalendarEventDevice(Entity):
```

Comment:
This should not be an entity. This will not be written to the state machine.

Reply:
So, what should I use ? just Object ?

Reply:
But why `CalendarEventDevice` is an entity ?